tests/test_models.py

```diff
@@ -1,270 +1,181 @@
-
-    predictor = CNNPatchPredictor(
-        predefined_model=predefined_model,
-        pretrained_weight=pretrained_weight,
-        batch_size=1,
-    )
-    # don't run test on GPU
-    output = predictor.predict(
-        dataset,
-        return_probabilities=return_probabilities,
-        return_labels=return_labels,
-        on_gpu=on_gpu,
-    )
-    predictions = output["predictions"]
-    if return_probabilities:
-        probabilities = output["probabilities"]
-        assert len(probabilities) == len(predictions)
-    if return_labels:
-        labels = output["labels"]
-        assert len(labels) == len(predictions)
-
-    if return_probabilities:
-        for idx, probabilities_ in enumerate(probabilities):
-            probabilities_max = max(probabilities_)
-            assert (
-                np.abs(probabilities_max - probabilities_check[idx]) <= 1e-8
-                and predictions[idx] == predictions_check[idx]
-            )
-
-    return save_dir_path
-
-
-def _test_outputs_api3(
-    dataset,
-    backbone,
-    return_probabilities=True,
-    return_labels=True,
-    probabilities_check=None,
-    predictions_check=None,
-    num_classes=9,
-    on_gpu=False,
-):
-    """Helper function to get the model output using API 3."""
-    # API 3
-    model = CNNPatchModel(backbone=backbone, num_classes=num_classes)
+ON_GPU = True
 
-    # coverage setter check
-    model.set_preproc_func(lambda x: x - 1)  # do this for coverage
-    assert model.get_preproc_func()(1) == 0
-    # coverage setter check
-    model.set_preproc_func(None)  # do this for coverage
-    assert model.get_preproc_func()(1) == 1
 
-    predictor = CNNPatchPredictor(model=model, batch_size=1, verbose=False)
-    # don't run test on GPU
-    output = predictor.predict(
-        dataset,
-        return_probabilities=return_probabilities,
-        return_labels=return_labels,
-        on_gpu=on_gpu,
+def _get_temp_folder_path():
+    """Return unique temp folder path"""
+    new_dir = os.path.join(
+        rcParam["TIATOOLBOX_HOME"], f"test_model_patch_{int(time())}"
     )
-    predictions = output["predictions"]
-    if return_probabilities:
-        probabilities = output["probabilities"]
-        assert len(probabilities) == len(predictions)
-    if return_labels:
-        labels = output["labels"]
-        assert len(labels) == len(predictions)
+    return new_dir
 
 
 def test_create_backbone():
     """Test for creating backbone."""
     backbone_list = [
         "alexnet",
-        "resnet18",
-        "resnet34",
-        "resnet50",
-        "resnet101",
-        "resnext50_32x4d",
-        "resnext101_32x8d",
-        "wide_resnet50_2",
-        "wide_resnet101_2",
-        "densenet121",
-        "densenet161",
-        "densenet169",
-        "densenet201",
-        "googlenet",
-        "mobilenet_v2",
-        "mobilenet_v3_large",
-        "mobilenet_v3_small",
+        # "resnet18",
```

Comment:
Why are these commented out?

Reply:
Uncommented now, likely I locally tested sthg and forgot to add them back.